### ISSUE TITLE
cursor styles don't appear to be working properly

### ISSUE THREAD

**Issue by iamcodemaker** [tailwindcss/tailwindcss#1804]
I'm seeing this on my site and on the [documentation examples](https://tailwindcss.com/docs/cursor/#app) in both firefox and chrome. The proper cursor style will appear briefly then go back to the default pointer.

**Comment by estevanmaito** [comment]
Could you share the version of your Firefox and Chrome?

You can type `navigator.userAgent` in the Dev Tools Console, and paste the result here.

**Comment by iamcodemaker** [comment]
Chrome:
```
"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36"
```

Firefox:
```
"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:77.0) Gecko/20100101 Firefox/77.0"
```

**Comment by iamcodemaker** [comment]
This appears to be a false alarm, it's working now. I suspect it is something specific to some software I have running. Sorry for the noise.

**Comment by estevanmaito** [comment]
That's why I asked if it was happening everywhere.

It seems to be a system wide problem, that some software in the background affects cursors in the browser, especially with macOS. Most notably Adobe ones. Sometimes it can be "solved" closing the app or restarting the system.

Did you have something in particular running in the background that you think could trigger this? It could help people searching in the future, not just for Tailwind.

**Comment by iamcodemaker** [comment]
I had Serato DJ Pro running at the time, but I can't reproduce the issue, so I'm not sure if that was it. I'll keep an eye out for this again and circle back if I see it.

**Comment by Snake-Sanders** [comment]
I landed here for the same issue, not a big deal, I just want to give a feedback on what I saw.
In Brave Browser it works fine, only in Firefox there is this strange behaviour where the cursor  does not change to the hand when I hover a link. 
But I noticed that as soon as I right click, showing menu and then closing it, the cursor detects the link and shows the hand pointer, it kind of fixes the issue temporarily. Once I click on the link, the issue appears again.

Firefox 128.0.3 (64-bit)
MacBook pro M3
MacOS 14.2 (23C64)
